driver/helpers.c

```diff
@@ -0,0 +1,50 @@
+#include "helpers.h"
+#include <stdlib.h>
+#include <string.h>
+#include <stdio.h>
+
+#ifdef _WIN32
+int setenv(const char *name, const char *value, const int overwrite) {
+    if (overwrite != 0 && getenv(name) != NULL) return -1;
+    const size_t n = strlen(name) + 1 /* = */ + strlen(value) + 1 /* \0 */;
+    char *env = calloc(n, sizeof(char));
```

Comment:
It might be better to use VLA to reduce the heap allocation cost since setenv is always at the tail of the call chain.